tools/packaging/guest-image/build_image.sh

```diff
@@ -41,15 +41,18 @@ build_initrd() {
 	export AGENT_INIT="yes"
 	# ROOTFS_BUILD_DEST is a Make variable
 
-	if [ -z "${AA_KBC}" == "offline_sev_kbc" ]; then
```

Comment:
Yes that is the only problem so that fix works. My reasoning had been looking ahead to a potential SNP module needing to get copied. But we can cross that when we get there since our first priority is just getting it working. I'll switch to this.

Reply:
Hi @Alex-Carter01 , I was looking at the logic to build the initrd for SEV and I think it would be better to use `${TEE:-} == "SEV"` here.  My reasoning is that we want to add the kernel modules (i.e. `efi_secrets` and any friends) in the initrd regardless of the KBC type. As you already mentioned, later we will need to add a case for `online_sev_kbc` and who knows if in the future we come up with yet another KBC type.

Makes sense?